pliers/transformers.py

```diff
@@ -152,13 +150,32 @@ def __hash__(self):
         return hash(self.name + str(dict(zip(self._log_attributes, tr_attrs))))
 
 
-class BatchTransformerMixin(object):
+class BatchTransformerMixin(Transformer):
```

Comment:
Might have missed it, but we should probably add a test or two that test this class as directly as possible (e.g., with a dummy test class that inherits from `BatchTransformerMixin`).